worker/sshserver/manifold.go

```diff
@@ -65,33 +83,31 @@ func (config ManifoldConfig) startWrapperWorker(context dependency.Context) (wor
 		return nil, errors.Trace(err)
 	}
 
-	var stTracker workerstate.StateTracker
-	if err := context.Get(config.StateName, &stTracker); err != nil {
+	var apiCaller base.APICaller
+	if err := context.Get(config.APICallerName, &apiCaller); err != nil {
 		return nil, errors.Trace(err)
 	}
 
-	statePool, err := stTracker.Use()
-	if err != nil {
-		_ = stTracker.Done()
-		return nil, errors.Trace(err)
-	}
-
-	sysState, err := statePool.SystemState()
+	client, err := config.NewFacadeClient(apiCaller)
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
 
 	w, err := config.NewServerWrapperWorker(ServerWrapperWorkerConfig{
-		SystemState:     sysState,
 		NewServerWorker: config.NewServerWorker,
 		Logger:          config.Logger,
+		FacadeClient:    client,
 	})
 	if err != nil {
-		_ = stTracker.Done()
 		return nil, errors.Trace(err)
 	}
 
-	return common.NewCleanupWorker(w, func() {
-		_ = stTracker.Done()
-	}), nil
+	return w, nil
+}
+
+// NewFacadeClient returns a facade client for the SSH server worker to use.
+// This is more of a shim as the client returns a concrete type and we're really
+// just using this for testing.
+func NewFacadeClient(caller base.APICaller) (FacadeClient, error) {
```

Comment:
Don't need this, just use `apisshserver.NewClient` directly

Reply:
removed and added to the mock for bestfacadeversion